types/autoprefixer/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for autoprefixer 9.7
+// Type definitions for autoprefixer 10.0.1
```

Comment:
```suggestion
// Type definitions for autoprefixer 10.0
```
no patch is supported on DT:
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library